docs-client/src/containers/MethodPage/DebugPage.tsx

```diff
@@ -589,6 +595,13 @@ class DebugPage extends React.PureComponent<Props, State> {
       debugResponse,
     });
   }
+
+  private moveScroll() {
+    let scrollNode = this.scrollRef.current;
```

Comment:
We prefer `const` as much as possible. In this case, we can use `const` because there is no chance to change `scrollNode` reference.

Reply:
I'll change it. Thank you for review!